.github/workflows/csharp-app-release.yml

```diff
@@ -0,0 +1,217 @@
+      
+      - name: Restore NuGet Packages
+        run:   
+          msbuild /p:Configuration=Release /p:Platform=x64 /t:restore
+
+      - name: Build
+        id: build
+        run: msbuild /p:Configuration=Release /p:Platform=x64
+
+      - name: Test
+        run: vstest.console.exe ${{ inputs.test_files }} /Platform:x64
+
+      - name: Upload Release
+        id: upload
+        uses: actions/upload-artifact@master
+        with:
+          name: ${{ needs.release-checks.outputs.artifact_id }}
+          path: ${{ needs.release-checks.outputs.artifact_id }}/bin/Release/
+
+      - name: Delete release branch if deploy failed and fail
+        if: steps.build.outcome == 'failure' || steps.upload.outcome == 'failure'
+        run: |
+          git push origin -d release
+          echo "There was an error in one of the steps above."
+          exit 1
+        shell: bash
+
+  create-release:
+    needs: [deploy, release-checks]
+    runs-on: ubuntu-latest
+    container: zepben/pipeline-basic
+    env:
+      GITHUB_TOKEN: ${{ secrets.CI_GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ env.GITHUB_TOKEN }}
+
+      - name: Work around git permission issue
+        run: |
+          dname=$(echo ${{github.repository}} | cut -d'/' -f2)
+          git config --global --add safe.directory /__w/$dname/$dname
+        shell: sh
+
+      - name: Merge and Tag
+        id: merge
+        run: |
+          git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
+          git fetch --all
+          git branch -a
+          git merge origin/release
+          git push origin ${GITHUB_REF/refs\/heads\//}
+          git tag "v${{ needs.release-checks.outputs.version }}"
+          git push --tags
+          echo "::set-output name=tag::$(echo v${{ needs.release-checks.outputs.version }})"
+        shell: bash
+        continue-on-error: true
+
+      - name: Fail if any previous step failed
+        if: steps.merge.outcome == 'failure'
+        run: |
+          git push origin -d release
+          echo "There was an error in merging the branch. release branch was deleted."
+          exit 1
+        shell: bash
+
+      - name: Download binary
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ needs.release-checks.outputs.artifact_id }}
+          path: built-artifacts
+        continue-on-error: true
+
+      - name: Archive Release
+        uses: thedoctor0/zip-release@master
+        with:
+          type: zip
+          filename: ${{ needs.release-checks.outputs.artifact_id }}.zip
+          directory: built-artifacts
+
+      - name: Stat zip file
+        run: |
+          ls
+          echo "Y"
+          ls built-artifacts
+        shell: bash
+
+      - name: Create Release and upload assets
```

Comment:
It appears you're not updating the release-notes.md file. It seems like the test repo has that file but doesn't have docusaurus docs. Can you either make this update that file or switch this to using --grow-changelog on the version update? Although if you switch we'll have to re-do the existing release-notes.mds into the growing format.